ts/nix.ts

```diff
@@ -163,6 +170,21 @@ export function nixStrLit(
   };
 }
 
+/**
+ * Returns a `NixExpression` which represents a dependency of the file file
```

Comment:
```suggestion
 * Returns a `NixExpression` which represents a dependency of the file 
```

Though I don't understand this...

Reply:
Whoops, this was supposed to say `flake file`. The idea is that these flake dependencies are captured sort of like a side-effect in the AST to be pulled out at the end when generating the flake file. This solves a few problems, one of being able to declare flake file dependencies in your `garn.ts` (although this currently doesn't have a nice API yet), but the other is to remove the need to install gomod2nix and npmlock2nix for every garn project. Now they are only pulled in as needed